docs/reference/template-tags.md

```diff
@@ -49,14 +53,18 @@ The context options object is added after area tag arguments following the `with
 
 Context options are optional for all core and official Apostrophe widget types.
 
-## component
+::: note
+Context options are not the best place for most widget configuration. That should be done in the [area field configuration](/reference/field-types/area.md#widgets). Context options are used to supplement that with options that only apply to the specific template context. Context options cannot change which widgets are permitted in an area.
+:::
+
+## `component`
 
 The `component` tag inserts an asynchronous component into the template. See the [async components guide](/guide/async-components.md) for more on using this feature.
 
 ### Usage
 
 ```django
-{% component 'moduleName:componentName' with data %}
+{% component 'module:componentName' with data %}
```

Comment:
Not sure why we wouldn't want consistency, we don't say `component`, why do we say `module`?

Reply:
In the case of `componentName`, we're in the section about `component` and already used that term in this example (as the tag keyword). I'm not sure what my thinking was at the time, but I'm guessing I used `module` so this would be less verbose. It is explained below as well.